packages/core/admin/ee/server/routes/index.js

```diff
@@ -129,7 +136,7 @@ module.exports = [
     path: '/review-workflows/workflows',
     handler: 'workflows.find',
     config: {
-      middlewares: [enableFeatureMiddleware('review-workflows')],
+      middlewares: [enableFeatureMiddleware('review-workflows', 'admin.reviewWorkflows.enabled')],
```

Comment:
@remidej I think this might be useful for audit logs too. As routes here were only disabled by using the license feature flag, but I think we could use something like this to also check for the project config.